classes/TaskRunner/Miscellaneous/UpdateConfig.php

```diff
@@ -95,6 +108,40 @@ public function run()
         }
     }
 
+    /**
+     * @param array $parameters
+     */
+    public function inputCLIParameters($parameters)
+    {
+        $this->isRunInCLI = true;
+        $this->cliParameters = $parameters;
+    }
+
+    /**
+     * Fetch data from either $_REQUEST (web context) or given input (CLI context)
+     */
+    protected function fetchConfigurationData()
+    {
+        if ($this->isRunInCLI) {
+            return $this->getCLIParameters();
+        } else {
+            return $this->getRequestParams();
+        }
```

Comment:
You can remove the useless else after return
```suggestion
        if ($this->isRunInCLI) {
            return $this->getCLIParameters();
        }
        return $this->getRequestParams();
```